runtime/tests/interpreter/reference_test.go

```diff
@@ -1238,45 +1238,3 @@ func TestInterpretResourceReferenceInvalidationOnDestroy(t *testing.T) {
 
 	})
 }
-
-func TestInterpretReferenceTrackingOnInvocation(t *testing.T) {
```

Comment:
Maybe keep this test, and indirection to the reference (e.g: https://github.com/onflow/cadence/pull/2573#discussion_r1231219902), so it will go to the interpreter.

Reply:
I tried several ways, but always get `referenced resource has been moved or destroyed after taking the reference`. Could you maybe try? Maybe I'm missing something